<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMsgDetail</class>
 <widget class="QDialog" name="DialogMsgDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditMsgDetail"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxOperate">
     <property name="title">
      <string/>
     </property>
     <property name="qssGroup" stdset="0">
      <string>GroupBoxNoBorder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnReply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Reply</string>
        </property>
        <property name="qssGroup" stdset="0">
         <string>ToolButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnForwarding">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Forwarding</string>
        </property>
        <property name="qssGroup" stdset="0">
         <string>ToolButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMoveTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Move to</string>
        </property>
        <property name="qssGroup" stdset="0">
         <string>ToolButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRead">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Read</string>
        </property>
        <property name="qssGroup" stdset="0">
         <string>ToolButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
